### PR TITLE
LPS-61468 Uses the default icon for icon-menu "ellipsis-v"

### PR DESCRIPTION
**Pull request by ealonso**

### CODE REVIEW

**Comment by ealonso** [comment]
Hey @sergiogonzalez, @marcellutavares, @matethurzo, @drewbrokke, @pei-jung, @jbalsas, @juliocamarero

I've did some changes around all the portlets,

 - Removed icons from drop down menu
 - Added btn-lg css class to buttons
 - Updated liferay-ui:icon-menu with lexicon style 

Please check my changes,
Thanks!!

File: modules/apps/server-admin/server-admin-web/src/main/resources/META-INF/resources/captcha.jsp [hunk 1]
```diff
@@ -28,5 +28,5 @@
 </aui:fieldset>
 
 <aui:button-row>
-	<aui:button cssClass="save-server-button" data-cmd="updateCaptcha" value="save" />
+	<aui:button cssClass="brn-lg save-server-button" data-cmd="updateCaptcha" value="save" />
```
Comment (drewbrokke) [inline modules/apps/server-admin/server-admin-web/src/main/resources/META-INF/resources/captcha.jsp]: Looks like a typo here, @ealonso

File: modules/apps/staging/staging-configuration-web/src/main/resources/META-INF/resources/view.jsp [hunk 2]
```diff
@@ -288,7 +288,7 @@ boolean disableStagingOptions = GetterUtil.getBoolean(SessionMessages.get(lifera
 
 		<c:if test="<%= !privateLayoutSet.isLayoutSetPrototypeLinkActive() && !publicLayoutSet.isLayoutSetPrototypeLinkActive() && !disableStagingOptions %>">
 			<aui:button-row>
-				<aui:button cssClass="btn-primary" type="submit" />
+				<aui:button cssClass="btn-lg" cssClass="btn-primary" type="submit" />
```
Comment (drewbrokke) [inline modules/apps/staging/staging-configuration-web/src/main/resources/META-INF/resources/view.jsp]: Looks like an extra `cssClass` attribute here.

**Comment by drewbrokke** [comment]
Thanks for updating these, @ealonso!

**Comment by brianchandotcom** [comment]
@sharonchoi can you work with @ealonso to fix these poshi bugs? Or send it to the right person? Thx.

**Comment by sharonchoi** [comment]
@brianchandotcom ok, will look into them.

**Comment by evannagayama** [comment]
@ealonso it looks like there is actually a bug on this pull. If you look at the functional tests that failed for staging and adt (which uses staging) there is an error message, "Staging Configuration is temporarily unavailable".

**Comment by ealonso** [comment]
Please see: https://github.com/sharonchoi/liferay-portal/pull/278

**Pull request closed without merging**